### PR TITLE
Add health endpoint for agent transport

### LINKED ISSUE #2818: Document new `/health` API available in the backend websocket API
### Description
Document the new `/health` endpoint which will become available with https://github.com/sensu/sensu-go/pull/4075. This provides the same functionality as https://docs.sensu.io/sensu-go/latest/api/health/ but is accessible via the backend websocket API (a.k.a. the agent port, `8081` by default).

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [x] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
We may be able to add a snippet that says it's available on both the HTTP API and weboscket API to https://docs.sensu.io/sensu-go/latest/api/health/.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Documents the new /health API endpoint for the backend websocket API to check the status of the agent transport.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2818

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@amdprophet and @cwjohnston a few questions about this one...

- I used the [health API](https://docs.sensu.io/sensu-go/latest/api/health/) example as a starting point based on info in the [sensu-go PR](https://github.com/sensu/sensu-go/pull/4075) that sounded like this new endpoint does the same thing as GET /health except for 8081 instead of 8080.
- I'm not sure my description in the agent reference is correct -- I used "backend websocket health API" to distinguish this endpoint from the [health API](https://docs.sensu.io/sensu-go/latest/api/health/) endpoint, but our [list of ports](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/install-sensu/#ports) describes 8081 as the agent API. I know this new endpoint is distinct from [/healthz](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/#healthz-get), so I was trying to find a way to describe it differently but I'm not sure I have it right.
- Do you think I put this new endpoint in the right place in the agent reference? If this endpoint is actually the same as the health API but just uses a different port in the request URL, should it go in the health API doc instead?

**Comment by hillaryfraley** [comment]
I updated this PR based on feedback from Cameron re: the questions in my previous comment. I tested the GET http://127.0.0.1:8081/health request and the response body I received is identical to the GET http://127.0.0.1:8080/health response body, so I maintained the same examples. If I should be seeing something else in the GET http://127.0.0.1:8081/health response body, please let me know.

File: content/sensu-go/6.2/api/health.md [hunk 1]
```diff
@@ -86,3 +86,83 @@ output           | {{< code shell >}}
   ]
 }
 {{< /code >}}
+
+## Get health data for your agent transport
+
+The `/health` API endpoint provides HTTP GET access to health data for your Sensu agent transport via the backend websocket.
```
Comment (cwjohnston) [inline content/sensu-go/6.2/api/health.md:92]:
```suggestion
The `/health` API endpoint provides HTTP GET access to health data for your Sensu agent transport via the backend websocket.
Sensu backend `/health` API information is duplicated by this agent transport API endpoint as an affordance to satisfy the load balancing and security requirements of some deployments.
```

Comment (cwjohnston) [reply content/sensu-go/6.2/api/health.md:92]: Aiming to explain that this is duplicate information and suggest why you might want one over the other.

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**